ee/tabby-ui/app/(dashboard)/settings/general/components/security-form.tsx

```diff
@@ -241,9 +242,9 @@ export const GeneralSecurityForm = () => {
     )
   }
 
-  return data && !stale ? (
-    <SecurityForm defaultValues={defaultValues} onSuccess={onSuccess} />
-  ) : (
-    <FormSkeleton />
+  return (
+    <LoadingWrapper loading={fetching || stale} fallback={<FormSkeleton />}>
```

Comment:
cc @wwayne It might be possible to use `loading-wrapper` in the `job-list` or directly use the `fetching` to determine whether to display the skeleton screen, thereby removing useIsQueryInitialized.

Reply:
Got it, thanks. I will take a try later